src/postcssPlugin.js

```diff
@@ -46,13 +46,15 @@ export default postcss.plugin("stylelint", (options = {}) => {
           // Handle either ES6 or CommonJS modules
           pluginImport = pluginImport.default || pluginImport
           ;[].concat(pluginImport).forEach(plugin => {
+            /* istanbul ignore if */
             if (!plugin.ruleName) {
```

Comment:
This is not a deprecated condition, so I think it should contribute to code coverage stats. At any point somebody might try to load a plugin without a rule name.